glide.yaml

```diff
@@ -1,96 +1,170 @@
 package: github.com/cockroachdb/cockroach
 import:
-- package: github.com/Azure/go-ansiterm
-- package: github.com/Microsoft/go-winio
-- package: github.com/Sirupsen/logrus
-- package: github.com/StackExchange/wmi
 - package: github.com/VividCortex/ewma
 - package: github.com/abourget/teamcity
-- package: github.com/agtorre/gocolorize
 - package: github.com/backtrace-labs/go-bcd
 - package: github.com/biogo/store
+  subpackages:
+  - interval
+  - llrb
 - package: github.com/cenk/backoff
 - package: github.com/chzyer/readline
-- package: github.com/client9/misspell
 - package: github.com/cockroachdb/c-jemalloc
+  # Inherently risky.
+  version: 42e6a32cd7a4dff9c70d80323681d46d046181ef
 - package: github.com/cockroachdb/c-protobuf
+  subpackages:
+  - cmd/protoc
 - package: github.com/cockroachdb/c-rocksdb
-- package: github.com/cockroachdb/c-snappy
+  # Inherently risky.
+  version: b5ca031b93fde49bfa2ba99aba423136aebf3c06
 - package: github.com/cockroachdb/cmux
 - package: github.com/cockroachdb/cockroach-go
-- package: github.com/cockroachdb/crlfmt
+  subpackages:
+  - crdb
 - package: github.com/cockroachdb/pq
-- package: github.com/cockroachdb/stress
 - package: github.com/codahale/hdrhistogram
 - package: github.com/coreos/etcd
```

Comment:
if there was a separate repo for raft, i'd agree, but this one is usually harmless to update since it rarely contains updates to raft.